ethereum/test/test_eth_app.js

```diff
@@ -41,11 +48,14 @@ contract("EthApp", function (accounts) {
       const weiAmount = web3.utils.toWei("0.25", "ether");
 
       // Deposit Ethereum to the contract and get the logs of the transaction
-      const { logs } = await this.ethApp.sendETH(
+      const tx = await this.ethApp.sendETH(
         recipient,
+        true,
```

Comment:
Shall we add a test case for non incentivized as well?

Reply:
refactored tests, have separate tests for the apps and for channels, will push soon.